components/src/hardware-sim/Labware/LabwareRender.tsx

```diff
@@ -174,6 +178,9 @@ export const LabwareRender = (props: LabwareRenderProps): JSX.Element => {
         highlight={props.highlight}
         highlightShadow={props.highlightShadow}
         wellStroke={props.wellStroke}
+        tipStatusByWellName={props.tipStatusByWellName}
+        handleClickWell={props.handleClickWell}
+        selectedTipsByIndex={props.selectedTipsByIndex}
```

Comment:
destructuring props would be better

```suggestion
        tipStatusByWellName={tipStatusByWellName}
        handleClickWell={handleClickWell}
        selectedTipsByIndex={selectedTipsByIndex}
```